db/seeds/base/add_applications.rb

```diff
@@ -73,15 +73,53 @@
     # users with one withdrawn application each
     FactoryBot.create_list(
       :application,
-      4,
+      2,
       :withdrawn,
+      %i[accepted rejected].sample,
       :with_random_user,
       :with_random_work_setting,
-      :with_random_lead_provider_approval_status,
       :with_random_participant_outcome_state,
       lead_provider:,
       course: Course.all.sample,
       cohort: Cohort.all.sample,
     )
+
+    # users with one eligible for funded place application each (cohort funding_cap true)
+    FactoryBot.create_list(
+      :application,
+      2,
+      :eligible_for_funded_place,
+      :with_random_user,
+      :with_random_work_setting,
+      :with_participant_id_change,
+      lead_provider:,
+      course: Course.all.sample,
+    )
+
+    # users with one eligible for funded place application each (cohort funding_cap false)
+    FactoryBot.create_list(
+      :application,
+      2,
+      :eligible_for_funded_place,
+      :with_random_user,
+      :with_random_work_setting,
+      :with_participant_id_change,
+      lead_provider:,
+      course: Course.all.sample,
+      cohort: Cohort.where.not("start_year = ?", 2024).sample,
+    )
+
+    # users with one funded place false application each
+    FactoryBot.create_list(
+      :application,
+      2,
+      :eligible_for_funded_place,
+      :with_random_user,
+      :with_random_work_setting,
+      :with_participant_id_change,
+      lead_provider:,
+      course: Course.all.sample,
+      funded_place: false,
```

Comment:
can we ensure that the application is accepted to be able to set this value? also that the cohort has funding cap: true? I think something to revisit across all the setup is:
1) If an application is accepted/withdraw/deferred and it is in a funding cap true cohort, we set the funded place true/false dependent on the eligible for funding field
2) If an application is accepted/withdrawn/deferred and is not in a funded cap true cohort, the funded place value remains false

Happy to jump on a call if there are questions